.github/workflows/publish-release.yml

```diff
@@ -57,4 +60,4 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.BALLERINA_BOT_TOKEN }}
         run: |
           curl -fsSL https://github.com/github/hub/raw/master/script/get | bash -s 2.14.1
-          bin/hub pull-request --base main -m "[Automated] Sync master after "$VERSION" release"
+          bin/hub pull-request -m "[Automated] Sync master after "$VERSION" release"
```

Comment:
Same

Reply:
Yeah it's a mistake. Reverted the change in 9816e8b09177408b05fd126a08646c365fc87fda